src/testfile

```diff
@@ -1,16 +1,5 @@
-ori R1, R0, 24
-slt R1, R0, R2
-ori R2, R0, 124
-fld F2, 200(R0)
-loop: fld F0, 0(R1)
-fmul F0, F0, F2
-fld F4, 0(R2)
-fadd F0, F0, F4
-fsd F0, 0(R2)
-addi R1, R1, -8
-addi R2, R2, -8
-bne R1,$0, loop
-bne F31,$100, loop
+fld F2, 200(R1)
```

Comment:
This file is not needed. I thought I have deleted from my git tracking but it somehow sneaked its way into this commit. It was just for testing purposes. Please disregard.